### ISSUE TITLE
Clearing configs hangs minidsp-rs

### ISSUE THREAD

**Issue by conrad-nash** [3ll3d00d/beqdesigner#421]
I have BEQd beta 15 installed on a windows 10 machine which is also running minidsp-rs 0.0.3.
minidsp-rs 0.0.3 connects over the network to another 0.0.3 instance running as a service on an rpi3 which is connected to the minidsp 2x4HD.

When executing any function from the Bypass Minidsp dropdown the first time, it works. I can see the server running through all 10 slots and executing a clear.

When running any function as a second try it hangs. It doesn't need to be the same function, clear 1 works, clear 2 has the issue.

I get a response in the minidsp console:

````
Sent: SetConfig { config: 03, reset: true }
Recv: MaybeConfigChanged
````

and both apps fail. BEQD spins and eventually times out, minidsp-rs needs to have another command sent to it to respond. Sending another command from another terminal "unlocks" the minidsp service and BEQD gets a checkmark.

I'm not sure if this is a BEQD issue or a minidsp-rs issue.

What I do know is that if I use the `all` option from a command line I can run it multiple times without any issues.

````minidsp --tcp 192.168.86.40:5333 input 0 peq all bypass on````

````minidsp --tcp 192.168.86.40:5333 input 0 peq all clear````

**Comment by 3ll3d00d** [comment]
Difficult to see how it is a beqd issue (assuming the command sent is valid), it is basically like

```
for i in {0..9}
do
    minidsp --tcp 192.168.86.40:5333 input 0 peq ${i} bypass on
done
```
Use the equivalent for Windows cmd if required

so probably you can test that at the cli and see what happens

**Comment by 3ll3d00d** [comment]
I should add a timeout to the CMD though and probably I can use the all option instead too

**Comment by 3ll3d00d** [comment]
noticed I had left some test code in which ignores the return code on a bypass call so perhaps this is related to this (ignoring an error and continuing anyway leaves the minidsp app in some invalid state?)

**Comment by conrad-nash** [comment]
Just tried beta 16 but it behaves the same. I've narrowed it down to the config change and am able to replicate it from a command line, including locally. I'll raise an issue with minidsp-rs

Thanks

**Comment by mrene** [comment]
Fixed on my side, was introduced in 0.0.3, fixed in 0.0.4.
